src/main/java/org/opentripplanner/routing/fares/impl/Ride.java

```diff
@@ -1,10 +1,13 @@
-package org.opentripplanner.routing.impl;
+package org.opentripplanner.routing.fares.impl;
 
+import java.time.LocalTime;
+import java.time.ZonedDateTime;
 import org.opentripplanner.model.FeedScopedId;
 import org.opentripplanner.model.Stop;
 
 import java.util.HashSet;
 import java.util.Set;
+import org.opentripplanner.model.StopLocation;
 
 /**
  * A set of edges on a single route, with associated information. Used only in calculating fares.
```

Comment:
In the future, I'd like to get rid of this class, all operations where this is used, should be done on regular `Leg` objects

Reply:
I agree